src/sage/geometry/hyperplane_arrangement/arrangement.py

```diff
@@ -2171,9 +2206,10 @@ def poset_of_regions(self, B=None, numbered_labels=True):
 
         EXAMPLES::
 
+            sage: # needs sage.combinat
             sage: H.<x,y,z> = HyperplaneArrangements(QQ)
             sage: A = H([[0,1,1,1], [0,1,2,3]])
-            sage: A.poset_of_regions()                                                  # needs sage.combinat
+            sage: A.poset_of_regions()
```

Comment:
Let's not make changes like this. Keep things specific. Plus it also is unrelated to the ticket and is likely to cause unnecessary conflicts.

In general, for the future, please do **not** make large scale PEP8/lint/etc. changes that are unrelated to the PR. It really makes it much harder to review the code and is likely to create annoying trivial conflicts with other PRs.

Reply:
My bad, I misunderstood to be coherent with `# needs ...`. Should I keep it or not?

Reply:
No, please revert this change and all other similar changes that go from one line to an entire block. For the ones that are larger blocks (say, most or all lines in 4+ lines of testing code but you can use some judgement here), then it is better to keep them since you already did the work.

Reply:
I will take care in the next push.

Reply:
Basically done.